community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/functions/Length.scala

```diff
@@ -36,4 +40,17 @@ case object Length extends Function with SimpleTypedFunction {
 
   def asCommandExpression(invocation: ast.FunctionInvocation) =
     commandexpressions.LengthFunction(invocation.arguments(0).asCommandExpression)
+
+  override def semanticCheck(ctx: SemanticContext, invocation: FunctionInvocation): SemanticCheck =
```

Comment:
Same as above; return type annotation on this `def` but not on others in this file.